feature-libs/product-configurator/rulebased/cpq/cpq-configurator-normalizer.spec.ts

```diff
@@ -605,6 +626,20 @@ describe('CpqConfiguratorNormalizer', () => {
       );
     });
 
+    it('should return UIType NOT_IMPLEMENTED for CPQ DisplayAs INPUT and DataType differt from INPUT_STRING', () => {
+      const cpqAttr: Cpq.Attribute = {
+        pA_ID: 1,
+        stdAttrCode: 2,
+        displayAs: Cpq.DisplayAs.INPUT,
+        dataType: Cpq.DataType.INPUT_NUMBER,
+        isEnabled: true,
+        values: [],
+      };
+      expect(cpqConfiguratorNormalizer.convertAttributeType(cpqAttr)).toBe(
+        Configurator.UiType.NOT_IMPLEMENTED
+      );
+    });
```

Comment:
this does not seem to be related to 'move value from input field'. did you fix multiple issues with this PR?

Reply:
Yes. Several issues (directly or indirectly related to handling of input fields) are fixed with this pull request. All adjustments are mentioned in the pull request comment.